packages/sdk-webos/src/tasks/taskSdkConfigure.ts

```diff
@@ -7,7 +7,7 @@ export default createTask({
     isPrivate: true,
     fn: async () => {
         await checkAndConfigureWebosSdks();
-        await checkWebosSdk(true);
```

Comment:
since this param is removed, if the user doesn't have the sdk(I just changed the name of webOS_TV_SDK to anything else in /opt), it will throw an undescriptive error.
<img width="712" alt="Screenshot 2024-09-11 at 10 38 47" src="https://github.com/user-attachments/assets/9a4db3cd-1a1a-4d06-8e2a-3406ac58c604">
So I guess throwing an error for a missing SDK is only important, if the user is trying to run a webos app on a simulator, since the simulators are in webOS_TV_SDK/Simulator folder?

Reply:
fixed